releasewarrior/taskcluster.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+import aiohttp
+import argparse
+import asyncio
+import async_timeout
+import posixpath
+from taskcluster.async import Index
+from taskcluster.async import Queue
+
+from releasewarrior.wiki_data import get_current_build_index
+
+
+HG_URL = 'https://hg.mozilla.org/'
+
+REPOS = ('releases/mozilla-beta', 'releases/mozilla-release',
+         'releases/mozilla-esr52', 'mozilla-central')
+
+""" http://mozilla-version-control-tools.readthedocs.io/en/latest/hgmo/pushlog.html?highlight=json-pushes#json-pushes-command """  # noqa
+JSON_PUSHES = 'json-pushes?changeset={rev}&version=2&tipsonly=1&full=1'
+
+ACTION_INDEX = 'gecko.v2.{repo}.pushlog-id.{pushid}.actions'
+
+
+async def get_changeset_data(revision, session):
+    for repo in REPOS:
+        url = posixpath.join(HG_URL, repo, JSON_PUSHES.format(rev=revision))
+        async with async_timeout.timeout(10):
+            async with session.get(url) as response:
+                if response.status == 404:
```

Comment:
hm, I swear we knew stored the branch in json data.

I guess we could always call [get_release_info](https://github.com/mozilla-releng/releasewarrior-2.0/blob/cd5ee19aeff09f5b52bf33f83f203df244d7bb62/releasewarrior/wiki_data.py#L124) and make the repo list a dict with keys based on the branches that are returned from release.branch

However, this works too :)